### PR TITLE
Protobuff contracts added

### PR DESCRIPTION
**Pull request by Korgath**
## Description

Added EnergySupplierChangeAccepted, EnergySupplierChanged, EnergySupplierChangeCancelled contracts

## References

* #0000

### CODE REVIEW

**Review by erduna — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto:19]:
```suggestion
/**
 * This message is sent out when a energy supplier change is accepted. It informs who will be the future energy supplier of the respective metering point
 */
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+message EnergySupplierChangeAccepted {
+  string MeteringPointId = 1;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto:21]:
```suggestion
  string MeteringPointId = 1; // Unique metering point identification
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  string EnergySupplierGLN = 2;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto:22]:
```suggestion
  string EnergySupplierGLN = 2; // Unique Energy Supplier identifcation
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  string EffectiveDate = 3;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto:23]:
```suggestion
  string EffectiveDate = 3; // Date in which the supplier change will go into effect
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  string TransactionId = 4;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeAccepted.proto:24]:
```suggestion
  string TransactionId = 4; // Unique transaction ID of the process. Needed to effectuate the correct process at a later date.
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
+message EnergySupplierChangeCancelled {
+  string MeteringPointId = 1;
+  string TransactionId = 2;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto:22]:
```suggestion
  string TransactionId = 2; // Unique transaction ID of the process. Needed to cancel the correct process.
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
+message EnergySupplierChangeCancelled {
+  string MeteringPointId = 1;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto:21]:
```suggestion
  string MeteringPointId = 1; // Unique metering point identification
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
+message EnergySupplierChanged {
+  string MeteringPointId = 1;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto:21]:
```suggestion
  string MeteringPointId = 1; // Unique metering point identification
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  string EnergySupplierGLN = 2;
+  string EffectiveDate = 3;
+  string TransactionId = 4;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto:24]:
```suggestion
  string TransactionId = 4; // Unique transaction ID of the process. Needed to effectuate the correct accepted process.
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
+message EnergySupplierChanged {
+  string MeteringPointId = 1;
+  string EnergySupplierGLN = 2;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto:22]:
```suggestion
  string EnergySupplierGLN = 2; // Unique Energy Supplier identifcation.
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+  string EffectiveDate = 3;
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto:23]:
```suggestion
  string EffectiveDate = 3; // Date which the change of supplier goes into effect.
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChangeCancelled.proto:19]:
```suggestion
/**
 * This message is sent out when an awaiting change of energy supplier has been cancelled.
 */
```

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContracts";
+
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/Contracts/EnergySupplierChanged.proto:19]:
```suggestion
/**
 * This message is sent out when an energy supplier has been changed.
 */
```

**Review by erduna — CHANGES_REQUESTED** [review]
I have just added some comments documenting the contracts. Comments will be used for generating the .md later on.

**Review by erduna — APPROVED** [review]

**Review by cmgrd — APPROVED** [review]

**Pull request merged**